pkgs/development/compilers/crystal/default.nix

```diff
@@ -54,9 +54,13 @@ stdenv.mkDerivation rec {
   makeFlags = [ "CRYSTAL_CONFIG_VERSION=${version}"
                 "FLAGS=--no-debug"
                 "release=1"
-                "all" "docs"
```

Comment:
Hmm... why move `make docs` to `postBuild`?

Reply:
Because you couldn't `make docs` with the old crystal, so the new crystal had to be built first. We probably should disable paralell builds for this too.

Reply:
Or we can update the prebuilt binary version, right?

Reply:
exactly, that just wasn't available at the time.